benchmarking/README.md

```diff
@@ -0,0 +1,45 @@
+# WebAssembly Benchmarking Subgroup Charter
+
+The Benchmarking Subgroup is a sub-organization of the [WebAssembly Community
+Group](https://www.w3.org/community/webassembly/) of the W3C. As such, it is
+intended that its charter align with that of the CG. In particular, the sections
+of the [CG charter](https://webassembly.github.io/cg-charter/) relating to
+[Community and Business Group
+Process](https://webassembly.github.io/cg-charter/#process), [Contribution
+Mechanics](https://webassembly.github.io/cg-charter/#contrib),
+[Transparency](https://webassembly.github.io/cg-charter/#transparency), and
+[Decision Process](https://webassembly.github.io/cg-charter/#decision) also
+apply to the Subgroup.
+
+## Goals
+
+The mission of this subgroup is to provide a forum for collaboration on
+benchmarking efforts for WebAssembly programs, sharing best practices, tools,
+and knowledge related to benchmarking.
+
+## Scope
+
+The Subgroup will consider topics related to benchmarking of WebAssembly, including:
+
+- Best practices for benchmarking WebAssembly
+- Development and sharing of benchmarking tools
+- Knowledge sharing about existing benchmarks, systems, and projects
+- Public, high-level, recommendations about best practices, frameworks, suites, metrics, for WebAssembly benchmarking
+
+## Deliverables
```

Comment:
Clarification of the scope of this subgroup: As documented right now, it's mostly for discussion, best practices and tools - but I could also see a potential for discussions in the subgroup that could lead to amendments and extensions of existing proposals, or creation of new ones (resulting in better profiling/tracing for example), is that explicitly out of scope for this subgroup? If not, it might make sense to call out as a potential deliverable.